tests/test_theme_picker.py

```diff
@@ -6,25 +6,42 @@
 from shinyswatch._bsw5 import BSW5_THEME_NAME, bsw5_themes
 
 
+def get_theme_deps(theme: shiny.ui.Theme) -> htmltools.HTMLDependency:
+    if hasattr(theme, "_html_dependency"):
+        # shiny < v1.2.0
+        deps = getattr(theme, "_html_dependency")()
+    else:
+        deps = getattr(theme, "_html_dependencies")()
+        # In shiny >= v1.2.0, `_html_dependencies()` returns a list of deps, but for
+        # shinyswatch we're expecting the default case to be a single dependency.
+        assert isinstance(deps, list)
+        assert len(deps) == 1
+        deps = deps[0]
+
+    return deps
+
+
```

Comment:
Code is unused

```suggestion
```

Reply:
Or did you want to use this method within the package and then we can relax our shiny version requirement?

Reply:
Yeah, I originally had an implementation that let us relax that shiny version requirement that I took out on second thought. On the balance, I don't think it's worth maintaining the additional logic, which was repeated in a few places with slight variations because constraints and uses were different.

Reply:
removed unused code in cbcdb289737a29102ed8fb7c9345cab538184d0c